### PR TITLE
Maildrobe, Reinforced wooden table, and boxingdrobe fixes

### PR DESCRIPTION
**Pull request by Hyenh**
## About the PR 

Some stuff for maps.

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Hyenh
- add: Reinforced wooden table and Maildrobe

### CODE REVIEW

**Comment by Elijahrane** [comment]
you need to figure out some way to test locally or have someone help you finish this PR; I'm gonna be unavailable for a while

**Comment by Hyenh** [comment]
> you need to figure out some way to test locally or have someone help you finish this PR; I'm gonna be unavailable for a while

I'll try to find a way

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@
   - type: Advertise
     pack: DetDrobeAds
   - type: Sprite
-    sprite: Structures/Machines/VendingMachines/detdrobe.rsi
+    sprite: Structures/Machines/VendingMachines/DetDrobe.rsi
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml:63]: Why was this renamed? The file still appears to be `detdrobe.rsi` unless I missed something. Furthermore, if this is the drobe for the reporters, why does RepDrobeInventory have mailman equipment?

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml [hunk 2]
```diff
@@ -72,7 +72,6 @@
   - type: WiresVisuals
   - type: AccessReader
     access: [["Service"]]
-
 - type: entity
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml:75]: It's usually a good idea to keep these separated.
```suggestion

- type: entity
```

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml [hunk 3]
```diff
@@ -86,10 +85,34 @@
     layers:
     - state: off
       map: ["enum.VendingMachineVisualLayers.Base"]
-    - state: normal-unshaded
+    - state: off
       map: ["enum.VendingMachineVisualLayers.BaseUnshaded"]
       shader: unshaded
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml:90]: Wouldn't this make the entire vending machine unshaded?

File: Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Inventories/maildrobe.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+- type: vendingMachineInventory
+  id: RepDrobeInventory
+  startingInventory:
+    ClothingUniformJumpsuitMailman: 2
+    ClothingUniformJumpskirtMailman: 2
+    ClothingHeadHatsMailman: 2
+    ClothingBeltMailbag: 2
+    ClothingEarsHeadsetCargo: 2
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Inventories/maildrobe.yml:8]: RepDrobe suggests Reporter. Reporter or mailman?

File: Resources/Prototypes/Nyanotrasen/Recipes/Construction/Graphs/furniture/tables.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+              amount: 10
+              doAfter: 1
+	- node: TableWoodReinforced
+      entity: TableWoodReinforced
+      edges:
+        - to: TableFrame
+          completed:
+            - !type:SpawnPrototype
+              prototype: MaterialWoodPlank1
+              amount: 10
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Recipes/Construction/Graphs/furniture/tables.yml:42]: Is it really intended that this reinforced wooden table take 20 planks?

File: Resources/Prototypes/Nyanotrasen/Recipes/Construction/Graphs/furniture/tables.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
+- type: constructionGraph
+  id: Table
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Recipes/Construction/Graphs/furniture/tables.yml:2]: I believe this may conflict with the other Table constructionGraph, but I would have to check. A simple copy and paste may not work here.

**Review by Vordenburg — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Inventories/maildrobe.yml (hunk 4 above)
Comment (Hyenh) [reply Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Inventories/maildrobe.yml:8]: All of the repdrobe issues was because I was messing around and trying to figure out how vending machines worked in yaml. It seems I got sloppy with my cleanup.

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml (hunk 3 above)
Comment (Hyenh) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml:90]: I was confused about this too. I looked through all of the ss14 vending machine yaml, and they had similar states for their drobes. Once I figure out the issue with my IDE I'll actually tests my pr's to see if they work.

**Comment by Elijahrane** [comment]
would you like someone else to make these entities for you?

**Comment by Hyenh** [comment]
> would you like someone else to make these entities for you?

If that means someone making a pr of these, I would be grateful. There is something fundamentally wrong with my computer that doesn't allow me to test things. Also, if someone could pr the pibble sprites that would be great too.

**Comment by IamVelcroboy** [comment]
> If that means someone making a pr of these, I would be grateful. There is something fundamentally wrong with my computer that doesn't allow me to test things. Also, if someone could pr the pibble sprites that would be great too.

Uploaded the MailDrobe. I can do the table and pibbler as well soon.
